app/uk/gov/hmrc/customsservicestatusfrontend/controllers/DashboardController.scala

```diff
@@ -59,14 +60,14 @@ class DashboardController @Inject() (
         outage.endDateTime match {
           case Some(endDate) =>
             val end = endDate.atZone(ZoneId.of("Europe/London")).toLocalDate
-            !today.isBefore(start) && !today.isAfter(end)
+            today.isAfter(start) && today.isBefore(end)
```

Comment:
I think this will change the existing tested functionality and introduce a bug. We're comparing dates here rather than datetimes and the existing check is inclusive of start date and end date (ie. it returns true if today is the start date or the end date), but I think the altered check is exclusive.

You could instead simplify by changing from: 

` !today.isBefore(start) && !today.isAfter(end)`

to: 

!(today.isBefore(start) || today.isAfter(end))

Reply:
Would be good to add failing tests at the boundaries before making these corrections which confirm what the behaviour should be if same day (for both start and end).